EMMO/README.md

```diff
@@ -2,47 +2,71 @@
 This [W3ID](https://w3id.org) provides a persistent URI namespace for the [Elementary Multiperspective Material Ontology (EMMO)](https://github.com/emmo-repo/EMMO/) and its domain ontologies.
 
 ## Redirection Rules
-This section contains a general summary of the logic behind the redirection rules.  
+This section contains a general summary of the logic behind the redirection rules.
 
 1. `https://w3id.org/emmo`
```

Comment:
is it necessary to state this as `https://w3id.org/emmo#{NAME}` or is that already implied?

Reply:
I am open for suggestions. I skipped the `#{NAME}` part because it is stripped off by the server before any rules are applied.